packages/ui-settings/README.md

```diff
@@ -1,3 +1,36 @@
 # @polkadot/ui-settings
 
 Manages app settings including endpoints, themes and prefixes
+
+### Usage Example
+User preferences are set as a settings object in the browser's local storage.
+```
+import settings from '@polkadot/ui-settings';
+
+render () {
+  // get api endpoint for the selected chain
+  const WS_URL = settings.apiUrl();
+
+  // get the selected il8n language
+  const language = settings.il8nLang();
+
+  // get all available il8n languages
+  const languages = settings.availableLanguages();
+
+  // update settings
+  const updatedSettings = {
+    ...settings,
+    i18nLang: 'Arabic'
+  }
+  settings.set(updatedSettings);
+
+  // NOTE: API currently does not handle hot reconnecting properly,
```

Comment:
This is actually slightly wrong, the API now actually does - however, kept it as re-load for the UI since with themes, etc. just didn't want to "risk" it, i.e. it still assumes in places a certain "order to the universe".